### DISCUSSION TITLE
Malware, or future malware, DnSpy site

### DISCUSSION THREAD

**Discussion by JVimes** [dnSpyEx/dnSpy#353] [General]
`dnspy[dot]co` is my fourth Google hit. It's super shady: AI speak, no attribution, 30s delay on downloads. I assume this is (another) imposter who will bundle malware after a while. Just figured I'd post.

**Comment by ElektroKill** [comment]
There is sadly nothing I can do about the fake dnSpy websites. All you can do really is report them to google if they are actually malicious.

**Reply by JVimes** [reply]
Ya, totally. I understand. I'm just trying to think of things to do. If they're impersonating someone, Google might remove them. Or maybe their registrar. But we're at the mercy of the systems, I know.
